web/src/add_stream_options_popover.ts

```diff
@@ -31,36 +32,49 @@ export function initialize() {
             //  When showing the popover menu, we want the
             // "Add streams" and the "Filter streams" tooltip
             //  to appear below the "Add streams" icon.
-            const add_streams_tooltip = $("#add_streams_tooltip").get(0);
-            add_streams_tooltip._tippy?.setProps({
-                placement: "bottom",
-            });
-            const filter_streams_tooltip = $("#filter_streams_tooltip").get(0);
+            const add_streams_tooltip: ReferenceElement | undefined =
+                $("#add_streams_tooltip").get(0);
+            if (add_streams_tooltip) {
+                add_streams_tooltip._tippy?.setProps({
+                    placement: "bottom",
+                });
+            }
+            const filter_streams_tooltip: (ReferenceElement & HTMLElement) | undefined =
+                $("#filter_streams_tooltip").get(0);
             // If `filter_streams_tooltip` is not triggered yet, this will set its initial placement.
-            filter_streams_tooltip.dataset.tippyPlacement = "bottom";
-            filter_streams_tooltip._tippy?.setProps({
-                placement: "bottom",
-            });
+            if (filter_streams_tooltip) {
+                filter_streams_tooltip.dataset.tippyPlacement = "bottom";
+                filter_streams_tooltip._tippy?.setProps({
+                    placement: "bottom",
+                });
+            }
 
             // The linter complains about unbalanced returns
-            return true;
+            // eslint-disable-next-line consistent-return
+            return;
```

Comment:
I don't see why we shouldn't use the original `return true` here. 

What makes you think the new conditionals are the right way to handle the possibility of elements being not present? I'm not sure how you're deciding if those should be asserts, if they should return early if not present, or something else.

Reply:
I removed the original `return true` since tippy.js accepts the property `onShow` return type as `false | void`.  

I assumed that if conditionals are the right way to check whether the element is present or not.
Shall I use `minimalistic-assert` for this?

Reply:
Hmm, can you start a new thread in #frontend about the `onShow` return type? `false | void` is a bit surprising as a return type... but given that's an upstream decision, we probably should proceed with this.

Generally one should use `minimalistic-assert` if the situation is impossible, and an `if` statement if it is possible. So the question for you is: Is this a case that is possible?

Reply:
> Generally one should use `minimalistic-assert` if the situation is impossible, and an `if` statement if it is possible. So the question for you is: Is this a case that is possible?

I don't think it's possible, as these functions are called only after the Popover appears. I will use `minimalistic-assert` for this.